src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -86,6 +86,17 @@ public void Close()
             CloseWebSocket();
         }
 
+        public async Task WaitForClose(CancellationToken cancellationToken)
```

Comment:
```suggestion
        public async Task WaitForCloseAsync(CancellationToken cancellationToken)
```

NIT: based on other method names in the file probably should have suffix `async`.